modules/bank.js

```diff
@@ -0,0 +1,373 @@
+      let embed = u.embed()
+      .setAuthor(interaction.member.displayName, interaction.member.displayAvatarURL({dynamic: true}))
+      .addField("Amount", `${gb}${withdraw.value}\n$${withdraw.value / 100}`)
+      .addField("Balance", `${gb}${balance.balance - withdraw.value}`)
+      .setDescription(`**${Util.escapeMarkdown(interaction.member.displayName)}** just redeemed ${gb} for a store coupon code.`);
+      interaction.client.channels.cache.get(Module.config.channels.modlogs).send({embeds: embed});
+    } else {
+      interaction.reply({content: "Sorry, something went wrong. Please try again.", ephemeral: true});
+    }
+  } catch(e) { u.errorHandler(e, interaction); }
+}
+
+async function bankAward(interaction) {
+  try {
+    let giver = interaction.member;
+    let recipient = interaction.getMember("recipient", true);
+    if (recipient.id == giver.id) {
+      interaction.reply({content: `You can't award *yourself* ${ember}, silly.`, ephemeral: true});
+      return;
+    } else if (recipient.id == interaction.client.user.id) {
+      interaction.reply({content: `You can't award *me* ${ember}, silly.`, ephemeral: true});
+      return;
+    }
+    
+    let value = interaction.getInteger("amount", true);
+    if (value === 0) {
+       interaction.reply({content: "You can't award *nothing*.", ephemeral: true});
+       return;
+    }
+    value = value > 10000 ? 10000 : (value < -10000 ? -10000 : value);
+
+    let reason = interaction.getString("reason") || "Astounding feats of courage, wisdom, and heart";
+    
+    let award = {
+      currency: "em",
+      discordId: recipient.id,
+      description: `From ${giver.displayName} (House Points): ${reason}`,
+      value,
+      giver: giver.id,
+      hp: true
+    };
+    let receipt = await Module.db.bank.addCurrency(award);
+    let gbBalance = await Module.db.bank.getBalance(recipient.id, "gb");
+    let emBalance = await Module.db.bank.getBalance(recipient.id, "em");
+    let embed = u.embed()
+    .setAuthor(interaction.client.user.username, interaction.client.user.displayAvatarURL({dynamic: true}))
+    .addField("Reason", reason)
+    .addField("Your New Balance", `${gb}${gbBalance.balance}\n${ember}${emBalance.balance}`)
+    .setDescription(`${Util.escapeMarkdown(giver.displayName)} just ${value > 0 ?"awarded" : "docked"} you ${ember}${receipt.value}! This counts toward your House's Points.`);
+    recipient.send({embeds: embed});
+
+    interaction.reply(`${ember}${value} ${value > 0 ?"awarded to" : "docked from"} ${Util.escapeMarkdown(recipient.displayName)} for ${reason}`).then(u.clean);
```

Comment:
I'll have to look at `u.clean` - I forget if it will handle cleaning interaction replies.